ls1intum/Artemis
# `Plagiarism check`: Fix display of escaped characters

### Checklist
#### General
#### Client

### Motivation and Context
1) Closes #6120 
2) The Distribution chart gets displayed in reverse:
![grafik](https://user-images.githubusercontent.com/26540346/213863239-ada79d80-73c9-460f-9392-085b8e10142b.png)


### Description
1) #5991 introduced escaping for the plagiarism view. However, this makes the resulting text longer, and the provided start and end indices of JPlag were no longer valid. In some cases, this leads to a highlighted text starting in between an escaped character as seen in the issue.

2) JPlag 4.1.0 fixed this bug and now returns the correct order. Our workaround code is no longer needed. See https://github.com/jplag/JPlag/pull/794

### Steps for Testing
1) This is a bit tricky to reproduce since you need a JPlag match stating directly after an `>` character. 

2) Run a new plagiarism check and see that the chart displays the results correctly.

### Test Coverage
unchanged
